src/units/abilities.cpp

```diff
@@ -1963,6 +1965,15 @@ effect::effect(const unit_ability_list& list, int def, const_attack_ptr att, boo
 			}
 		}
 
+		if((type == "resistance")){
```

Comment:
Please remove the extra set of parenthesis.

Reply:
Done